doc/api/cli.md

```diff
@@ -1116,6 +1116,16 @@ Instructs Node.js to error prior to running any code if the policy does not have
 the specified integrity. It expects a [Subresource Integrity][] string as a
 parameter.
 
+### `--policy-required`
```

Comment:
While on the security-wg meeting, I realized that maybe `--require-policy` is a better naming than `--policy-required`. Just leaving this for discussion. @nodejs/security-wg

Reply:
No, because we already have `--require`. It might lead to confusion to have `--require-policy`, as in “the policy that controls `require` calls” instead what of I presume you mean, which is “the policy needs to be present.”

Reply:
That makes sense. Thanks for the clarification. Resolving my comment.

Reply:
You could use some other phrasing, like maybe `--need-policy` or something. Ideally we would have some consistent pattern across our flags for “must be defined.”

Reply:
Open to any naming convention desired.

Reply:
If no one has a better idea, I guess `--need-policy`? Let’s just pick something (that doesn’t include “require”) and land this.

Reply:
What about --enforce-policy?

Reply:
Policies are always enforced in the contexts they are loaded in, so I think `--need-*` as a convention might be better, it also would apply to things that are not "enforced".